packages/blockchain-wallet-v4-frontend/src/scenes/Login/template.tsx

```diff
@@ -446,67 +450,152 @@ const Login = (props: InjectedFormProps<{}, Props> & Props) => {
-                  showImage
+              <QRCodeContainer>
+                {props.secureChannelLoginState.cata({
+                  Success: () => {
+                    return (
+                      <Text size='14px' weight={600}>
+                        Success! Logging in...
+                      </Text>
+                    )
+                  },
+                  Failure: e => (
+                    <Text>{typeof e === 'string' ? e : 'Unknown Error'}</Text>
+                  ),
+                  Loading: () => {
+                    return (
+                      <Text size='14px' weight={600}>
+                        Please confirm the login on your mobile device...
+                      </Text>
+                    )
+                  },
+                  NotAsked: () => (
+                    <QRCodeWrapper
+                      value={qr_data}
+                      size={qr_data.length}
+                      showImage
+                    />
+                  )
+                })}
+              </QRCodeContainer>
+            </>
+          )}
+
+          {phonePubKey && (
+            <>
+              <CartridgeSentContainer>
+                <SuccessCartridge>
+                  <FormattedMessage
+                    id='scenes.login.wallet.message.sent'
+                    defaultMessage='Message Sent'
+                  />
+                </SuccessCartridge>
+              </CartridgeSentContainer>
+
+              <Text
+                size='16px'
+                color='grey900'
+                weight={600}
+                style={{ marginTop: '8px' }}
+              >
+                <FormattedMessage
+                  id='scenes.login.wallet.connected.title'
+                  defaultMessage='Mobile Device Connected'
                 />
-              )
-            })}
-          </QRCodeContainer>
+              </Text>
+
+              <Text
+                size='12px'
+                color='grey900'
+                weight={500}
+                style={{ marginTop: '8px' }}
+              >
+                <FormattedMessage
+                  id='scenes.login.wallet.connected.description_1'
+                  defaultMessage='We sent your connected mobile device a notification. Open the app to auto-log in on the web.'
+                />
+              </Text>
+              <Text
+                size='12px'
+                color='grey900'
+                weight={500}
+                style={{ marginTop: '24px' }}
+              >
+                <FormattedMessage
+                  id='scenes.login.wallet.connected.description_2'
+                  defaultMessage='Didn’t get the notification? Make sure you have push notifications enabled.'
+                />
+              </Text>
+
+              <TextGroup
+                inline
+                style={{ marginTop: '8px', lineHeight: '18px' }}
+              >
+                <Link size='12px' weight={500}>
```

Comment:
still checking with @matsjj which type of message we have to send in such case to socket

Reply:
ok we define how it should be.